src/bp-forums/activity.php

```diff
@@ -348,60 +344,6 @@ function activity_get_notification_permalink( $link, $item_id, $secondary_item_i
 			return $link;
 		}
 
-		/**
-		 * Render the activity content for discussion activity. 
-		 *
-		 * @since BuddyBoss 1.7.2
-		 * 
-		 * @param string $content  Activit content.
-		 * @param object $activity Activit data.
-		 *
-		 * @uses bbp_get_reply()           Get reply post data.
-		 * @uses bbp_get_topic_permalink() Get discussion permalink.
-		 *
-		 * @return string
-		 */
-		public function before_activity_content( $content, $activity ) {
-			global $activities_template;
-
-			// When the activity type does not match with the topic or reply.
-			if ( ! in_array( $activity->type, array( $this->topic_create, $this->reply_create ), true ) ) {
-				return $content;
-			}
-
-			// Set topic id when activity component is not groups.
-			if ( $this->component === $activity->component ) {
-				// Set topic id when activity type topic.
-				$topic_id = $activity->item_id;
-
-				// Set topic id when activity type reply.
-				if ( $this->reply_create === $activity->type ) {
-					$topic    = bbp_get_reply( $topic_id );
-					$topic_id = $topic->post_parent;
-				}
-			}
-
-			// Set topic id when activity component is groups.
-			if ( 'groups' === $activity->component ) {
-				// Set topic id when activity type topic.
-				$topic_id = $activity->secondary_item_id;
-
-				// Set topic id when activity type reply.
-				if ( $this->reply_create === $activity->type ) {
-					$topic    = bbp_get_reply( $topic_id );
-					$topic_id = $topic->post_parent;
-				}
-			}
-
-			// Topic.
-			$topic_permalink = bbp_get_topic_permalink( $topic_id );
-			$topic_title     = get_post_field( 'post_title', $topic_id, 'raw' );
-			$reply_to_text   = ( ! empty( $topic->ID ) && bbp_is_reply( $topic->ID ) ) ? sprintf( '<span class="bb-reply-lable">%1$s</span>', esc_html__( 'Reply to', 'buddyboss' ) ) : '';
-			$content         = sprintf( '<p class = "activity-discussion-title-wrap"><a href="%1$s">%2$s %3$s</a></p> <div class="bb-content-inr-wrap">%4$s</div>', esc_url( $topic_permalink ), $reply_to_text, $topic_title, $content );
-
-			return $content;
-		}
-
```

Comment:
Please don't remove this function

Reply:
Back to the previous one.
https://github.com/buddyboss/buddyboss-platform/pull/2635/commits/7593d31b8fde819594390a94b715746fd0abbee1